docs/user-guide.md

```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
 
    `./install-oracle.sh --help`
 
-1. Run installation. Only IP address of the first cluster node is required:
+1. Create cluster config file. The file `cluster_config.json` is provided as template
+   and should be updated accordingly.
+
+1. Run installation pointing to the file for planned cluster configuration:
```

Comment:
Suggestion:
"Install the database with the path to the cluster configuration file specified on the --cluster-config property:"